### PR TITLE
CSL-1188 Fix build without web and wallet

### PR DESCRIPTION
**Pull request by gromakovsky**

### CODE REVIEW

**Comment by gromakovsky** [comment]
@akegalj can you take a look at appveyor build?

In this PR I split `Pos.Wallet.Web.Error` into two modules and re-exported them from `Pos.Wallet.Web.Error`. Now appveyor fails with:
```
[00:34:04] ERROR in ./src/Daedalus/ClientApi.purs
[00:34:04] Module build failed: Error: purs-loader failed
[00:34:04]     at C:\w\daedalus\node_modules\purs-loader\lib\index.js:165:32
[00:34:04]     at Array.forEach (native)
[00:34:04]     at C:\w\daedalus\node_modules\purs-loader\lib\index.js:164:31
[00:34:04]     at tryCatcher (C:\w\daedalus\node_modules\bluebird\js\release\util.js:16:23)
[00:34:04]     at Promise._settlePromiseFromHandler (C:\w\daedalus\node_modules\bluebird\js\release\promise.js:512:31)
[00:34:04]     at Promise._settlePromise (C:\w\daedalus\node_modules\bluebird\js\release\promise.js:569:18)
[00:34:04]     at Promise._settlePromise0 (C:\w\daedalus\node_modules\bluebird\js\release\promise.js:614:10)
[00:34:04]     at Promise._settlePromises (C:\w\daedalus\node_modules\bluebird\js\release\promise.js:689:18)
[00:34:04]     at Async._drainQueue (C:\w\daedalus\node_modules\bluebird\js\release\async.js:133:16)
[00:34:04]     at Async._drainQueues (C:\w\daedalus\node_modules\bluebird\js\release\async.js:143:10)
[00:34:04]     at Immediate.Async.drainQueues (C:\w\daedalus\node_modules\bluebird\js\release\async.js:17:14)
[00:34:04]     at runCallback (timers.js:672:20)
[00:34:04]     at tryOnImmediate (timers.js:645:5)
[00:34:04]     at processImmediate [as _immediateCallback] (timers.js:617:5)
[00:34:04]  @ ./src/index.js 7:36-72
[00:34:04]  @ multi ./src/index.js
[00:34:04] 
[00:34:04] ERROR in [1/1 ModuleNotFound] 
[00:34:04] 
[00:34:04]   Module Pos.Wallet.Web.Error was not found.
[00:34:04]   Make sure the source file exists, and that it has been provided as an input to psc.
[00:34:04] 
[00:34:04]            Src   Lib   All
[00:34:04] Warnings   0     0     0  
[00:34:04] Errors     0     0     1  
[00:34:04] 
```

How to fix it?

File: src/node/Main.hs [hunk 1]
```diff
@@ -227,6 +237,8 @@ updateTriggerWorker = first pure $ worker mempty $ \_ -> do
 -- Wallet stuff
 ----------------------------------------------------------------------------
 
+#ifdef WITH_WALLET
+
```
Comment (Martoon-00) [inline src/node/Main.hs]: Why do you think e.g. `wallet` is not needed without `WITH_WALLET`? What about this https://github.com/input-output-hk/cardano-sl/pull/895/files#diff-816b9ae1342c0d73323326bc3391be25R129
Or `WITH_WEB` implies `WITH_WALLET`?

Comment (gromakovsky) [inline src/node/Main.hs]: >  Or WITH_WEB implies WITH_WALLET?

Hmm, no, it doesn't, but let's temporary assume it does 😄 

The task was to fix build w/o web **and** wallet and I think I did it. Perhaps it won't compile for web xor wallet, but it's anyway better than it was before.

Comment (Martoon-00) [inline src/node/Main.hs]: Hah, okay then

**Comment by gromakovsky** [comment]
@akegalj ping https://github.com/input-output-hk/cardano-sl/pull/895#issuecomment-305914301

**Comment by gromakovsky** [comment]
@akegalj ping again.

**Comment by akegalj** [comment]
@gromakovsky I am very sorry - my mailbox is full and not maintained last week. I will take a look at this now

**Pull request merged**